modules/common-attributes.adoc

```diff
@@ -55,7 +55,7 @@ endif::[]
 :osp: Red{nbsp}Hat OpenShift
 :olm-first: Operator Lifecycle Manager (OLM)
 :olm: OLM
-:rhacs-version: 4.5.1
```

Comment:
just curious: current latest version is 4.5.4, so I'm surprised this isn't either 4.5.0 or 4.5.4 (I don't really know how the docs repo branches out for patches)